packages/aws-cdk/lib/api/bootstrap-environment.ts

```diff
@@ -26,6 +27,12 @@ export interface BootstrapEnvironmentProps {
    * @default - the default KMS key for S3 will be used.
    */
   readonly kmsKeyId?: string;
+  /**
+   * Tags for cdktoolkit stack.
+   *
+   * @default - None.
+   */
+  tags?: Tag[];
```

Comment:
I'm thinking this should be called `stackTags` instead as it'd be ambiguous whether this refers to the bootstrap bucket or the toolkit stack.

what do you think?

Reply:
I used `tags` just to be consistent with `deploy-stack` options and cli arguments. I feel it's fine to leave it as `tags`. Let me know if otherwise

Reply:
tags is fine in the general sense... do tags on the stack propagate to the bucket?

Reply:
yes, the tags propagate to stack, confirmed it in the integration test